packages/vitest/src/node/plugins/index.ts

```diff
@@ -121,7 +121,7 @@ export async function VitestPlugin(
                 // by default Vite resolves `module` field, which not always a native ESM module
                 // setting this option can bypass that and fallback to cjs version
                 mainFields: [],
```

Comment:
No idea about `mainFields`. Should we respect it from user config as well?

```suggestion
                mainFields: viteConfig.resolve?.mainFields || [],
```